src/components/PageSections/Menu/_FullSiteMenu.scss

```diff
@@ -249,44 +253,88 @@
   line-height: $coa-line-height-xsmall;
 }
 
-.coa-ThemesNav__search-container-icon {
-  display: none;
-  position: absolute;
-  right: 0;
-  padding-right: $coa-space-level4;
-  font-size: $coa-font-size-medium;
-  .coa-ThemesNav__search-icon {
-    color: $coa-color-white;
-    padding-top: $coa-space-level1;
-    vertical-align: top;
+.coa-Header__right-controls-wrapper {
+  order: 3;
+  align-items: center;
+  justify-content: flex-end;
+  margin-right: 2rem;
+  padding-right: 2rem;
+
+  @include media($coa-medium-screen) {
```

Comment:
Let's keep the convention of using the mixins we've got for responsive styling. Use `@include tablet-and-desktop` for sizes larger than tablet, use `@include tablet` for styles only on tablet, etc.

Reply:
Is it a convention? I've been using the $coa-medium-screen because that what I saw more of. I think we have both everywhere.

Reply:
Ok yeah, I'm looking at our code now. I didn't realize how much we were using `@include media($coa-medium-screen)`. That's clearly the winner.

A pitch for using mixins going forward:
- they're more expressive, the names say exactly what they target
- since they're basically classes defined in one place, we can change the rules around tablet/mobile/desktop breakpoints in one place instead of 200.

But that being said, I don't want to try to make them a thing if they're not a thing. And they are clearly outnumbered by `@include media(*px)`. There is room for Janis to implement consistent standards on many fronts. But this is not a battle that needs to be waged right now, as this pattern is yet to be decided. 

I retract my mixin change requests. I don't want to make ya'll do busywork that actually goes against the design convention we're predominantly using.

Reply:
Yes. the css conventions could use a clean-up at some point. A lot of this code was moved around, and made it more obvious where some conventions of yesteryear had clashed with our current way of doing it. I'll go through and match what you caught. easy fix!